lib/helper/rules/PricerRule.js

```diff
@@ -0,0 +1,183 @@
+// Copyright 2019 OpenST Ltd.
```

Comment:
Change declaration type of the variable to `const` instead of `let`.

Please add method for generating executable data of `pay` method of PricerRule contract.